packages/client/modules/userDashboard/components/OrgBilling/DowngradeModal.tsx

```diff
@@ -0,0 +1,219 @@
+import {Close} from '@mui/icons-material'
+import {Checkbox} from '@mui/material'
+import React, {useState} from 'react'
+import DialogContainer from '../../../../components/DialogContainer'
+import DialogContent from '../../../../components/DialogContent'
+import DialogTitle from '../../../../components/DialogTitle'
+import PlainButton from '../../../../components/PlainButton/PlainButton'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {TeamBenefits} from '../../../../utils/constants'
+import DowngradeToStarterMutation from '../../../../mutations/DowngradeToStarterMutation'
+import {useFragment} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import useMutationProps from '../../../../hooks/useMutationProps'
+import SendClientSegmentEventMutation from '../../../../mutations/SendClientSegmentEventMutation'
+import {DowngradeModal_organization$key} from '../../../../__generated__/DowngradeModal_organization.graphql'
+
+const StyledDialogContainer = styled(DialogContainer)({
+  padding: 8
+})
+
+const StyledDialogTitle = styled(DialogTitle)({
+  fontSize: 24,
+  lineHeight: '32px'
+})
+
+const StyledDialogContent = styled(DialogContent)({
+  display: 'flex',
+  flexDirection: 'column',
+  paddingBottom: 16
+})
+
+const LabelGroup = styled('div')({
+  paddingTop: '32px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  width: '100%'
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_900,
+  fontSize: 16,
+  lineHeight: 1.5,
+  padding: '24px 24px 0px 24px'
+})
+
+const ActionLabel = styled('div')({
+  color: PALETTE.SKY_500,
+  fontSize: 16,
+  lineHeight: 1.5,
+  fontWeight: 600,
+  '&:hover': {
+    cursor: 'pointer'
+  }
+})
+
+const UL = styled('ul')({
+  margin: 0
+})
+
+const LI = styled('li')({
+  fontSize: 16,
+  lineHeight: '32px',
+  color: PALETTE.SLATE_900,
+  textTransform: 'none',
+  fontWeight: 400,
+  textAlign: 'left'
+})
+
+const ButtonRow = styled(PlainButton)({
+  width: '100%',
+  display: 'flex',
+  flexWrap: 'nowrap',
+  alignItems: 'center'
+})
+
+const Label = styled('div')({
+  paddingLeft: 8,
+  fontSize: 16,
+  lineHeight: '32px',
+  color: PALETTE.SLATE_900,
+  width: '100%'
+})
+
+const StyledCloseButton = styled(PlainButton)({
+  height: 24,
+  position: 'absolute',
+  right: 16,
+  top: 16
+})
+
+const CloseIcon = styled(Close)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  '&:hover,:focus': {
+    color: PALETTE.SLATE_800
+  }
+})
+
+const StyledCheckbox = styled(Checkbox)({
+  svg: {
+    fontSize: 28
+  },
+  width: 28,
+  height: 28,
+  textAlign: 'center',
+  userSelect: 'none'
+})
+
+type Props = {
+  closeModal: () => void
+  organizationRef: DowngradeModal_organization$key
+}
+
+const reasonsToLeave = [
+  'Parabol is too expensive',
+  'Budget changes',
+  'Missing key features',
+  `Not using Parabol's paid features`,
+  'Moving to another tool (please specify)'
```

Comment:
Do we plan to add textfield with the tool name?

Reply:
Ah great point! Updated 👍